### ISSUE TITLE
can you make a binary release built with new Optix 7.4 SDK?

### ISSUE THREAD

**Issue by oscarbg** [DeclanRussell/NvidiaAIDenoiser#20]
to see if there are any perf improvements vs 2019 build..

**Comment by DeclanRussell** [comment]
Yes I plan to do a release, however I am hitting some regression in quality compared to the previous release when using the albedo and normal AOVs. Once I have solved this, I want to add support for a couple more features that were exposed in OptiX 7.3, then I will ship a new release. In terms of performance though I haven't seen too much of an improvement, the denoiser is in the Nvidia driver after all so both versions tap into the same thing. However, the new features exposed in 7.3 are much more exciting than performance IMHO, so stay tuned! 😄

**Comment by alejobrainz** [comment]
Hi Declan.  Do you have any updates regarding this release?  Thanks!

**Comment by oscarbg** [comment]
now it means a Optix 7.4 build..

**Comment by mojo-jojo-7** [comment]
Hey Declan! Are you still considering doing this?
